app/models/ability.rb

```diff
@@ -187,6 +188,7 @@ def signed_in_with_cfp_role(user)
     # ids of all the conferences for which the user has the 'cfp' role
     conf_ids_for_cfp = Conference.with_role(:cfp, user).pluck(:id)
 
+    can [:index, :show, :update], Resource, conference_id: conf_ids_for_cfp
```

Comment:
`:index` and `:show` can be both replaced by `:read`: https://github.com/ryanb/cancan/wiki/Action-Aliases

There are some more places were this appear, please change it there too 🙂